packages/mobile/src/utils/trackUtils.ts

```diff
@@ -1,13 +1,10 @@
-import type { Track } from '@audius/common'
+import type { ID, UserTrackMetadata } from '@audius/common'
 
-export const isAvailableForPlay = (
-  track: Pick<
-    Track,
-    'is_available' | 'is_delete' | 'is_invalid' | 'is_unlisted' | 'owner_id'
-  >,
-  currentUserId: number
-) =>
-  track.is_available &&
-  !track.is_delete &&
-  !track.is_invalid &&
-  !(track.is_unlisted && track.owner_id !== currentUserId)
+export const isTrackValid = (track: UserTrackMetadata, currentUserId: ID) => {
```

Comment:
minor, but imo isValid feels pretty generic and to me sounds more like a check for illegal values. Not sure the best name, but I think isAvailableForPlay was actually closer

Reply:
yeah i like that, how about "downloadable"?